examples/basic_cog.py

```diff
@@ -0,0 +1,78 @@
+import discord
+from discord.ext import commands
+
+# https://discordpy.readthedocs.io/en/latest/ext/commands/cogs.html is the documentation page for cogs.
+description = '''An example bot to showcase the discord.ext.commands extension
+module.
+There are a number of utility commands being showcased here.
+The example uses cogs to organize the code
+'''
+
+intents = discord.Intents.default()
+intents.members = True
+
+# We define the basic bot, setting the prefix, description and intents.
+bot = commands.Bot(command_prefix='?', description=description, intents=intents)
+
+# We define the class that will house the Miscellaneous commands.
+# The class inherits from the `commands.Cog` class.
+# The main advantage is we can take advantage of Object Oriented Programming.
+# This also groups all commands in the help command under the `Misc` cog.
+# The name kwarg passed allows us to set the name of the cog. If not set it defaults to the name of the class.
+class Miscellaneous(commands.Cog, name="Misc"):
+    """
+    A simple Miscellaneous cog.
+    """
+
+    def __init__(self, bot):
+
+        # We pass in the bot parameter when initializing the class, and make it a property. This lets us access the bot parameter throughout.
+        self.bot = bot
+        # You can also add other properties to the class.
+        self.other_value = 'This is a property of the class.'
```

Comment:
```suggestion
        self.other_value = 'This is a class attribute.'
```
Not a property here.